content/blog/2024/09/chunking-dsl/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Unlocking Efficient Data Processing with the Chunking DSL"
+date: 2024-09-04
+draft: false
+authors: [orpiske]
+categories: ["Releases"]
```

Comment:
It would be great if you can add the article to the category "AI" as well. So we can find it a lot easier later when working on an AI project.

Reply:
Good idea. Let me do that!